### DISCUSSION TITLE
Texture floodfill problem

### DISCUSSION THREAD

**Discussion by yoye21** [ImageMagick/ImageMagick#3521] [Help]
I want floodfill specific pattern in specific color.

like : 
![image](https://user-images.githubusercontent.com/82192912/114121928-4168de80-992a-11eb-83f1-e9717d925011.png)

**Comment by snibgo** [comment]
Your question is vague.

You can create an image that contains a black square with gray borders, and a second image that contains a white square with gray borders, then duplicate and append the results as required.
